src/client/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -131,9 +132,31 @@ public partial interface IPublicClientApplication : IClientApplicationBase
         /// in order to override the default authority set at the application construction. Note that the overriding authority needs to be part
         /// of the known authorities added to the application construction.
         /// </remarks>
+        [Obsolete("This method is deprecated. Please use AcquireTokenByUsernamePassword(IEnumerable<string> scopes, string username, string password) instead.", true)]
```

Comment:
Let's soft-deprecate, i.e. recommend people to use string, but not block them if they prefer to continue to use SecureString.

Reply:
Reason: ".NET no longer recommends using SecureString and MSAL would anyway need to put the plaintext value of the password on the wire, as required by the OAuth protocol. See https://docs.microsoft.com/en-us/dotnet/api/system.security.securestring?view=net-5.0#remarks"